lib/astc_encode.cpp

```diff
@@ -807,3 +890,232 @@ ktxTexture2_CompressAstc(ktxTexture2* This, ktx_uint32_t quality) {
 
     return ktxTexture2_CompressAstcEx(This, &params);
 }
+
+struct decompression_workload
```

Comment:
I don't like having the decoder in a file called `astc_encode.cpp`. I know we talked about this before but I don't recall the answer to "is there a lot of code in this file shared between encode and decode?" If not then I'd prefer the decoder in a separate file. If there is then consider renaming the file to `astc_codec.cpp`.

Reply:
> is there a lot of code in this file shared between encode and decode?
The answer was/is yes. I think better rename the file.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/952/commits/8f4b5a65c9fc29f67b1cf14be36802dbfa4f5358